pysages/ssages/methods/abf.py

```diff
@@ -41,7 +41,7 @@ def _abf(snapshot, cv, grid, N, helpers):
     dims = grid.shape.size
     natoms = np.size(snapshot.positions, 0)
     get_grid_index = build_indexer(grid)
-    indices, momenta = helpers
+    indices, momenta, *_ = helpers
```

Comment:
this is the part, where a namedtuple or dict, wouldn't need refactoring of old code.
(the additional fields would just be ignored.)